authz/client/pom.xml

```
@@ -18,6 +18,12 @@
     <description>Keycloak Authz: Client API. This module is supposed to be used just in the Keycloak repository for the testsuite. It is NOT supposed to be used by the 3rd party applications.
         For the use by 3rd party applications, please use `org.keycloak:keycloak-authz-client` module.</description>
 
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.release>8</maven.compiler.release>
```

If I recall correctly setting the `release` should be enough to get compatible byte-code for a certain API level. There should be no need to set the `source` to `8`.

So best case scenario:
```suggestion
        <maven.compiler.release>8</maven.compiler.release>
```

And at worst:
```suggestion
        <maven.compiler.target>8</maven.compiler.target>
        <maven.compiler.release>8</maven.compiler.release>
```

---

yes, you are right. It's done for the four poms now.